rubicon/java/api.py

```diff
@@ -104,6 +105,19 @@ def convert_args(args, type_names):
             jarg = java.NewByteArray(len(arg))
             java.SetByteArrayRegion(jarg, 0, len(arg), arg)
             converted.append(jarg)
+        elif isinstance(arg, Sequence) and type_name[0] == ord(b'['):
+            if type_name[1] == ord(b'I'):
+                jarg = java.NewIntArray(len(arg))
+                java.SetIntArrayRegion(jarg, 0, len(arg), (jint * len(arg))(*arg))
+                converted.append(jarg)
+            elif type_name[1] == ord(b'F'):
+                jarg = java.NewFloatArray(len(arg))
+                java.SetFloatArrayRegion(jarg, 0, len(arg), (jfloat * len(arg))(*arg))
+                converted.append(jarg)
+            elif type_name[1] == ord(b'D'):
+                jarg = java.NewDoubleArray(len(arg))
+                java.SetDoubleArrayRegion(jarg, 0, len(arg), (jdouble * len(arg))(*arg))
+                converted.append(jarg)
```

Comment:
Is there any reason to not include byte, char, bool, and any other primitive for which there is a SetXArrayRegion method?

Reply:
Bytes are already smoothly handled by the `b'xyz'` handling a few lines before this.

`char` and `bool`... sure... but I just haven't really seen 'em in the wild, so I'm just being lazy.

I can try adding them.

Reply:
For `char` arrays, I want to punt. For the rest, they're now handled.